### PR TITLE
Add dcp_lion to library, elevated railways to `ceqr_survey`

### PR DESCRIPTION
**Pull request by fvankrieken**
Part of #486, #575 

Resulting table
image

A bit interesting that things like airtrain are included. There are 56 distinct segments, so might be worth making sure that all of them are appropriate. See row_type description in [lion metadata](https://s-media.nyc.gov/agencies/dcp/assets/files/pdf/data-tools/bytes/lion_metadata.pdf) (page 32)

### CODE REVIEW

File: dcpy/library/ingest.py
Comment (fvankrieken) [inline dcpy/library/ingest.py:92]: I feel like this is a decent check, in general our outputs assume one layer so we shouldn't be allowing multiple layers going through. Might raise errors if some of our input datasets do have multiple layers and by some convention we're just getting one of them. Would appreciate thoughts

**Comment by fvankrieken** [comment]
Hmm. Archival of lion [just failed](https://github.com/NYCPlanning/data-engineering/actions/runs/7936606882/job/21672252919). I got this error locally, then it resolved, so I assumed it was something that I had inadvertantly fixed, but seems that it might be happening inconsistently? Should poke around a bit before merging

File: dcpy/library/ingest.py
Comment (damonmcc) [reply dcpy/library/ingest.py:92]: yea since we're always trying to output one table, seems good to error if multiple are found and none specified

**Review by damonmcc — APPROVED** [review]
LGTM but not sure if that parquet issue is resolved

**Comment by fvankrieken** [comment]
Put off investigation lion parquet issue, wrote up in #600

**Pull request merged**
